var/spack/repos/builtin/packages/cuda-toolkit/package.py

```diff
@@ -165,6 +165,24 @@ class Cuda(Package):
     # see https://docs.nvidia.com/cuda/archive/11.4.0/cuda-gdb/index.html#release-notes
     depends_on('ncurses abi=5', type='run', when='@:11.3.99+dev')
 
+    provides('cuda@11.6')
```

Comment:
```suggestion
```

Reply:
Will this autodetect the version, or should I still have the version mapping table?

Reply:
I think it's actually better to delete this line